packages/extension/src/utils/format.test.ts

```diff
@@ -156,6 +156,148 @@ describe('formatFlags', () => {
     };
     expect(formatFlags(flags, 'NFTokenMint')).toBe('Transferable');
   });
+
+  it('should format OfferCreate flags correctly when given as a number', () => {
+    const flags = 0x00010000 | 0x00020000; // both tfPassive and tfImmediateOrCancel flags are set
```

Comment:
Really minor, but could be imported from xrpl.js

Reply:
It's from create offer, it's a bad rebase

Reply:
oh I see it's the test file, ok i will change it